Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -1044,17 +1044,19 @@ final class KSRAnalyticsTests: TestCase {
     let segmentClient = MockTrackingClient()
     let ksrAnalytics = KSRAnalytics(dataLakeClient: dataLakeClient, segmentClient: segmentClient)
 
+    let typeContext = KSRAnalytics.TypeContext.init(params: DiscoveryParams.recommendedDefaults)
```

Comment:
I don't think the `.init` part of the line is needed. Try to remove it in all the places you added it in and see if it compiles.

Reply:
Also I don't think you need this constant since you are only using it on line 1052.